src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -183,6 +183,13 @@ object Parameters {
     "internal_execute_query_in_sync_mode"
   )
 
+  // Internal option to use AWS region URL. By default, it is 'true'.
+  // This option may be removed without any notice in any time.
+  // Introduced since Spark Connector 2.9.3
```

Comment:
Is this statement true?
1, why this option is internal but not public?
2, why will we remove it?

Reply:
This option is used to give a way for customer to disable this change if there is regression.
When this fix is confirmed to work well in customer environment. It can be removed.